README.md

```diff
@@ -0,0 +1,190 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Bookstore\] ](#-bookstore-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo and other links ](#-live-demo-and-other-links-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Bookstore] <a name="about-project"></a>
+
+> The bookstore is an MVP version of a previously built app, awesome books. It is a Single Page App (SPA) that allows users to:
+
+- Display a list of books.
+- Add a book.
+- Remove a selected book.
+
+## 🛠 Built With <a name="built-with"></a>
+
+The major tools and languages used are:
+
+- REACT
+- JavaScript
+<!-- Features -->
+### Tech Stack <a name="tech-stack"></a>
+
+> The following tech stack were used
+
+<details>
+  <summary>REACT</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">REACT</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The web app has the following key features:
+
+- **[Allows you to display a list of books]**
+- **[Add a book]**
+
+<p align="right">[<a href="#readme-top">Back to Top</a>]</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo and other links <a name="live-demo"></a>
+
+> Here, we will paste links as per the submission instructions.
+
+- [Live Demo Link]()
+
+<p align="right">[<a href="#readme-top">Back to Top</a>]</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> In order to run this project you need:
+
+- Node js installed on your computer
+- Npm Node package manager
+- Terminal
+
+### Setup
+
+Clone this repository to your desired folder:
+
+> https://github.com/krisbobo/maths-magicians.git
+
```

Comment:
- [ ] Great work setting up your _**README.md**_ file. The link that you provided for people to clone this repository is that of the **_Math Magician_** project. Kindly include the correct link. 👍

Reply:
Noted👍